platzky/attachment/core.py

```
@@ -0,0 +1,261 @@
+    filename: str
+    content: bytes
+    mime_type: str
+
+    @classmethod
+    def from_bytes(
+        cls,
+        content: bytes,
+        filename: str,
+        mime_type: str,
+        max_size_override: int | None = None,
+    ) -> "AttachmentProtocol": ...
```

Ruff (0.14.13)

Remove quotes from type annotation

Remove quotes

(UP037)